static/style.css

```diff
@@ -246,3 +246,97 @@ p.text-black.text-base.company-blurb {
 a.underline, span.underline {
     text-underline-offset: 3px;
 }
+
+#people-page-grid {
+    display: grid;
+    grid-template-columns: repeat(auto-fit, minmax(240px, 1fr));
+    grid-auto-rows: 135px;
+    grid-auto-flow: dense;
+    gap: 10px;
+    margin-top: 30px;
+    margin-bottom: 30px;
+}
+
+#people-page-grid > * {
+    grid-area: span 1 / span 1;
+}
+
+#people-page-grid .tile-2x2 {
+    grid-area: span 2 / span 2;
+}
+
+@media (max-width: 470px) {
+    #people-page-grid .tile-2x2 {
+        grid-area: span 2 / span 1;
+    }
+}
+
+#people-page-grid .tile-1x2 {
+    grid-area: span 2 / span 1;
+}
+
+.person-tile {
+    justify-self: stretch;
+    background-color: #f3f3f3;
+    max-height: 135px;
+    min-width: 240px;
```

Comment:
We should not size the elements with pixels. Use percentage or some other relative sizing unit to give dimensions to elements.
This becomes important when we have different-sized screens on desktops, tablets, and mobiles.

Same comment for other CSS styles.